.github/workflows/security-scan.yml

```
@@ -11,9 +11,24 @@
   ZAP_VERSION: 2.15.0
 
 jobs:
+  jobs:
+  detect-environments:
+    runs-on: ubuntu-latest
+    outputs:
+      environments: ${{ steps.environments.outputs.result }}
+    steps:
+      - uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)